Extensions/CameraViewport/cameraviewportruntimeobject-pixi-renderer.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * The PIXI.js renderer for the DummyRuntimeObject.
+ *
+ * @class DummyRuntimeObjectPixiRenderer
+ * @constructor
+ * @param {gdjs.CameraViewportObject} runtimeObject The object to render
+ * @param {gdjs.RuntimeScene} runtimeScene The gdjs.RuntimeScene in which the object is
+ */
+gdjs.CameraViewportObjectPixiRenderer = function(runtimeObject, runtimeScene)
+{
+    this._object = runtimeObject; // Keep a reference to the object to read from it later.
+    this._renderer = runtimeScene.getGame().getRenderer().getPIXIRenderer();
+    this._renderTexture = new PIXI.RenderTexture.create({width: runtimeObject.getWidth(), height: runtimeObject.getHeight()});
+    this._sprite = new PIXI.Sprite(this._renderTexture);
+    this._scene = runtimeScene;
+};
+
+gdjs.CameraViewportObjectRenderer = gdjs.CameraViewportObjectPixiRenderer; //Register the class to let the engine use it.
+
+gdjs.CameraViewportObjectPixiRenderer.prototype.getRendererObject = function() {
+    return this._sprite;
+};
+
+gdjs.CameraViewportObjectPixiRenderer.prototype.update = function() {
+    /* 
+     * Remove all cameras: cameras can't render themselves as it could result in an endless loop.
+     * We could theoretically count the renders and replace the camera with a blank texture after too many renders,
+     * But chrome web gl automatically blocks textures that are made from a texture to be applied to the texture it is made from.
+     */
+     this._scene._constructListOfAllInstances();
+    for(var objectId in this._scene._allInstancesList) {
+        var object = this._scene._allInstancesList[objectId];
+        if(object.type !== "CameraViewport::CameraViewport") continue;
+        this._scene
+            .getLayer(object.layer)
+            .getRenderer()
+            .removeRendererObject(object.getRendererObject());
```

Comment:
Interesting :) Do you think you could do something like setting `visible` on the `object.getRendererObject()`(that we know to be a `DisplayObject`) to false, and then put it back to its initial value?
Might be faster than removing the renderer object and then adding it back, which are fairly costly in comparison.

Reply:
> Do you think you could do something like setting visible  


Good idea! Works perfectly :)